index.html

```diff
@@ -180,7 +180,38 @@ <h3> <img src="/vectorsImg/Tertiary/Icons/circle.svg" alt=""> Skills</h3>
                 </section>
             </article>
         </section>
+
+        <section class="contact-section">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+            <form action="https://formspree.io/f/mqkngqpl" method="post">
+                <ul>
+                    <li>
+                        <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                    </li>
+                    <li>
+                        <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                    </li>
+                    <li>
+                        <textarea name="user_msg" id="msg" maxlength="500" placeholder="Enter text here" required></textarea>
+                    </li>
+                    <li class="btn-container">
+                        <button type="submit">Get in touch</button>
+                    </li>
+                </ul>
+            </form>
+        </section>
     </main>
+    <footer>
+        <ul>
+            <li><a href="#"><i class="bi bi-github"></i></a></li>
+            <li><a href="#"><i class="bi bi-linkedin"></i></a></li>
+            <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
+            </li>
+            <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+            <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
+        </ul>
```

Comment:
- [ ] Great job using a list to layout the icons, however, the styling is not like the design, kindly fix that.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/165382293-dde0bcb1-cbda-4fff-8ae6-b8a4159feb44.png)


### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/165382408-3c5500ee-eec4-44fd-be9e-7cb9c0dbad0c.png)